document/js-api/index.bs

```diff
@@ -1776,13 +1778,13 @@ In practice, an implementation may run out of resources for valid modules below
 <li>The maximum number of operands to `array.new_fixed` is 10,000.</li>
 </ul>
 
-An implementation must throw a {{RuntimeError}} if one of the following limits is exceeded during runtime:
+An implementation must fail to grow tables or memories beyond the following limits at runtime.
 In practice, an implementation may run out of resources for valid modules below these limits.
 
 <ul>
-<li>The maximum size of a table is 10,000,000.</li>
-<li>The maximum size of a 32-bit memory is 65,536 pages (4 GiB).</li>
-<li>The maximum size of a 64-bit memory is 262,144 pages (16 GiB).</li>
+<li>A table can grow to have up to 10,000,000 entries.</li>
+<li>A 32-bit memory can grow to have up to 65,536 pages (4 GiB).</li>
```

Comment:
Again this limit doesn't mean anything; you can't have a 32-bit memory larger than 4GB. (I realize this one was already present, but I think it should be removed.)